python/neutron-understack/neutron_understack/neutron_understack_mech.py

```diff
@@ -166,19 +161,13 @@ def delete_network_postcommit(self, context):
         if provider_type == p_const.TYPE_VXLAN:
             conf = cfg.CONF.ml2_understack
             ucvni_group = conf.ucvni_group
-            try:
-                self.nb.ucvni_delete(network_id)
-            except Exception as e:
-                LOG.exception(
-                    "unable to delete network %(net_id)s", {"net_id": network_id}
-                )
-                raise exc.NetworkNotFound(net_id=network_id) from e
-
+            self.nb.ucvni_delete(network_id)
```

Comment:
Does this not need to be wrapped in a neutron_lib exception to work correctly (e.g. throw back a useful error instead of a 500)

Reply:
fixed in nautobot client by subclassing the NeutronException